packs/classfeatures/ruffian.json

```diff
@@ -87,6 +87,12 @@
                 "path": "system.skills.itm.rank",
                 "value": 1
             },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "system.proficiencies.defenses.medium.rank",
+                "value": "min(@actor.system.proficiencies.defenses.light.rank,3)"
+            },
```

Comment:
Instead of trying to be clever we should do this exactly as written. Set the proficiency to trained on the item then 2 REs, predicated on having the stated features.